src/main/java/org/wirvsvirus/locoronando/category/CategoryController.java

```diff
@@ -0,0 +1,52 @@
+package org.wirvsvirus.locoronando.category;
+
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/v1/category")
+public final class CategoryController {
+  private final CategoryRepository categoryRepository;
+  private final CategoryService categoryService;
+
+  @Autowired
+  private CategoryController(
+    CategoryRepository categoryRepository,
+    CategoryService categoryService
+  ) {
+    this.categoryRepository = categoryRepository;
+    this.categoryService = categoryService;
+  }
+
+  @PostMapping(path = "add")
```

Comment:
Why du you add "add" to the path? In classic REST we do not take the action in the path - it's the action (POST) already :)

Reply:
Oh, okay ;)